### PR TITLE
qa: test also with stakeholder-managers

### PR DESCRIPTION
**Pull request by darosior**
We had a bug that went unnoticed because we were only testing setups with well-separated roles.

### CODE REVIEW

File: tests/test_framework/revault_network.py [hunk 1]
```diff
@@ -30,58 +30,96 @@ def __init__(
...
         self.man_wallets = []
 
         self.csv = None
 
-    def deploy(self, n_stakeholders, n_managers, csv=None):
+    def deploy(self, n_stakeholders, n_managers, n_stkmanagers=0, csv=None):
         """
         Deploy a revault setup with {n_stakeholders} stakeholders, {n_managers}
-        managers and optionally fund it with {funding} sats.
+        managers.
```
Comment (danielabrozzoni) [inline tests/test_framework/revault_network.py:43]: `, and {n_stkmanagers} stakeholders_managers` ?

Comment (darosior) [reply tests/test_framework/revault_network.py:43]: meh. It's self explanatory, i should remove it entirely if i retouch

**Comment by darosior** [comment]
Rebased after #170 merge

**Comment by danielabrozzoni** [comment]
I ran `test_sigfetcher` multiple times in local and it keeps failing :(

**Comment by darosior** [comment]
Ok, "miscomitted" :sweat_smile: . I was missing a part of the diff in the last commit, should be good now!

**Comment by danielabrozzoni** [comment]
ACK 1a1ad393d746841c3b1215f75045d9f5f55d737d

**Pull request merged**